src/Components/ServiceScene/LogsPanelScene.tsx

```diff
@@ -371,7 +372,20 @@ export class LogsPanelScene extends SceneObjectBase<LogsPanelSceneState> {
         ]);
     }
 
-    return panel.build();
+    const vizPanel = panel.build();
+    vizPanel.setState({
+      titleItems: (
+        <JsonRootNodeNavigation
+          sceneRef={this}
+          showEndingSeparator={false}
+          hideIfEmpty={true}
+          forceRender={true}
+          maxWidth={config.featureToggles.logsPanelControls ? `calc(100vw - 320px)` : `calc(100vw - 610px)`}
+        />
+      ),
+    });
+
+    return vizPanel;
```

Comment:
Do we want the navigation in the Logs Panel? It doesn't change the log line viewing. Would be cool to have it below the controls `Show Original Log Line`, `Newest First` etc.

<img width="1512" height="117" alt="Image" src="https://github.com/user-attachments/assets/004a8e73-ec80-4e3b-8142-bc87b96fe2f6" />